File: Entrada de Dados/Numerica/numero.jl
function main()

  NUMERO = 0
  NUMERO_AUX = ""

  while true
    print("Entre um numero: ")
    NUMERO_AUX = readline()
    if tryparse(Int, NUMERO_AUX) !== nothing
      NUMERO = parse(Int, NUMERO_AUX)
      break
    else
      println("Entrada invalida. Por favor, tente novamente.")
    end
  end
  println("Voce informou o numero: ", NUMERO)
end

main()

File: Entrada de Dados/Numerica/numero.cs
using System;

class Program
{
  static void Main()
  {
    
    int NUMERO;
    string NUMERO_AUX;
    
    do
    {
      Console.Write("Entre um numero: ");
      NUMERO_AUX = Console.ReadLine();
      if (int.TryParse(NUMERO_AUX, out NUMERO))
      {
        break;
      }
      else
      {
        Console.WriteLine("Entrada invalida. Por favor, tente novamente.");
      }
    } while (true);
    Console.WriteLine("Voce informou o numero: " + NUMERO);
  }
}
